lib/src/main/java/bio/terra/javapfb/Library.java

```diff
@@ -0,0 +1,45 @@
+package bio.terra.javapfb;
+
+import bio.terra.common.logging.LoggingInitializer;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.fasterxml.jackson.module.paramnames.ParameterNamesModule;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.retry.annotation.EnableRetry;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+@SpringBootApplication(
+    scanBasePackages = {
+      // Scan for logging-related components & configs
+      "bio.terra.common.logging",
+      // Scan for tracing-related components & configs
+      "bio.terra.common.tracing",
+      // Scan all service-specific packages beneath the current package
+      "bio.terra.javapfb"
+    })
+@ConfigurationPropertiesScan("bio.terra.javapfb")
+@EnableRetry
+@EnableTransactionManagement
+@EnableConfigurationProperties
+public class Library {
+  public static void main(String[] args) {
+    new SpringApplicationBuilder(Library.class).initializers(new LoggingInitializer()).run(args);
+  }
+
+  public Library() {}
+
+  @Bean("objectMapper")
+  public ObjectMapper objectMapper() {
```

Comment:
Personally I would put bean providers in a separate class than your main app to separate them from the app definition, for example: https://github.com/DataBiosphere/terra-data-catalog/blob/main/common/src/main/java/bio/terra/catalog/config/BeanConfig.java

If you don't use spring (and still need ObjectMaper) it can go into a static field.

Reply:
Ok, I've removed spring and will keep this in mind if I need to add ObjectMapper in the future.